0800-appendix-58067-gramophone.Rmd

```diff
@@ -209,14 +216,19 @@ Electrofishing was conducted in 2022 with results summarised in Tables
 \@ref(tab:tab-fish-site-58067) - \@ref(tab:tab-fish-dens-58067) and
 Figure \@ref(fig:plot-fish-box-58067).
 
-In 2023 fish sampling was conducted below the observed falls using dip nets. Six Coho were caught, 2 fry (FL ranging from 65-67mm), and 4 par (FL ranging from 70-90mm), see photos \@ref(fig:fish). Since Coho were captured below the falls but none at crossing `r as.character(my_site)` (during the 2022 sampling), it is presumed that the falls are a barrier to Coho passage. 
+In 2023 fish sampling was conducted below the observed falls using dip nets. Six Coho parr were captured and had a wide range of sizes (FL ranging from 65-90 mm) which suggests that spawning may have occurred at different times or rearing environments were more favourable (more food, warmer) for some, see figures \@ref(fig:fish1), \@ref(fig:fish2), \@ref(fig:fish3). Since Coho were captured below the falls but none at crossing `r as.character(my_site)` during the 2022 sampling, it is presumed that the falls are a barrier to Coho passage. 
 
 
-```{r fish, fig.cap = "Coho fry and par captured below the falls with dip nets during 2023 sampling."}
+```{r fish1, fig.cap = "Coho parr captured below the falls with dip nets during 2023 sampling."}
 knitr::include_graphics("data/photos/58067/2023/IMG_2843_k_fish.JPG")
+```
+
+```{r fish2, fig.cap = "Coho parr captured below the falls with dip nets during 2023 sampling."}
```

Comment:
might be some complexity here I don't see but seems like if we put all the photos in a directory called 2023 (ex. `data/photos/2023/58067/IMG_2843_k_fish.JPG` we could `rsync` over the entire 2023 directories from onedrive to the repos more easily as `name` changes happen only on onedrive over time.  

`_fish` is an ambiguous name that can break `fpr_photo_pull_by_str` if there is another file with that same partial string (due to `stringr::str_subset(., str_to_pull)`).  All good in that run but being more explicit helps save errors (been through it a bunch and its annoying and time consuming).

``` r
# this will result in an error if they are both in the same file
str_subset('fish.JPG', 'fish')
#> Error in str_subset("fish.JPG", "fish"): could not find function "str_subset"
str_subset('fish1.JPG', 'fish')
#> Error in str_subset("fish1.JPG", "fish"): could not find function "str_subset"


# to avoid the chance of errors use
str_subset('f2.JPG', 'f2')
#> Error in str_subset("f2.JPG", "f2"): could not find function "str_subset"
str_subset('f1.JPG', 'f1')
#> Error in str_subset("f1.JPG", "f1"): could not find function "str_subset"
```

<sup>Created on 2024-04-23 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>


No need to redo but worth considering for the rest of the sites